### PR TITLE
RHIDP-428 Send PR test results to ReportPortal

### PR DESCRIPTION
**Pull request by zdrapela**
## Description

Add Data Router (`droute`) to Dockerfile. Send PR test results to ReportPortal.

## Which issue(s) does this PR fix

- Fixes https://issues.redhat.com/browse/RHIDP-428

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing

### CODE REVIEW

File: .ibm/pipelines/openshift-ci-tests.sh [hunk 1]
```diff
@@ -251,7 +308,7 @@ check_and_test() {
     echo "Backstage is not running. Exiting..."
     exit 1
   fi
-    run_tests $namespace
+    run_tests $release_name $namespace
```
Comment (josephca) [inline .ibm/pipelines/openshift-ci-tests.sh:311]: @zdrapela minor cosmetic issue :) - indentation may need to be fixed

**Review by josephca — APPROVED** [review]
/lgtm
/approve

**Comment by subhashkhileri** [comment]
@zdrapela , do we have separate ReportPortal dashboards for nightly runs and PR runs, So we can track success and failure rates for both types of jobs?

Are you currently distinguishing between nightly runs and PR check runs in ReportPortal?

File: .ibm/pipelines/openshift-ci-tests.sh (hunk 1 above)
Comment (zdrapela) [reply .ibm/pipelines/openshift-ci-tests.sh]: Thank you! Fixed 😉 Please add LGTM again after that

**Comment by zdrapela** [comment]
> @zdrapela , do we have separate ReportPortal dashboards for nightly runs and PR runs, So we can track success and failure rates for both types of jobs?
Dashboards are not configured yet, but I plan to work on that soon.

> Are you currently distinguishing between nightly runs and PR check runs in ReportPortal?
Yes, I'm adding labels in ReportPortal. PR check will get labels `job_type:presubmit` and e.g. in this case`pr:1208`. Nightly is not yet configured, but I'll work on that also...

**Review by rnapoles-rh — APPROVED** [review]
/lgtm

**Pull request merged**